yb-voyager/src/tgtdb/yugabytedb.go

```diff
@@ -1329,3 +1353,26 @@ type NodeMetrics struct {
 	Status  string
 	Error   string
 }
+
+func (yb *TargetYugabyteDB) checkIfCurrentUserIsSuperUser() (bool, error) {
+	// SELECT rolname, rolsuper FROM pg_roles WHERE rolname='yugabyte';
+	query := "SELECT rolsuper FROM pg_roles WHERE rolname=current_user"
+	rows, err := yb.Query(query)
+	if err != nil {
+		return false, fmt.Errorf("querying if user is superuser: %w", err)
+	}
+	defer rows.Close()
+
+	var isSuperUser bool
+	if rows.Next() {
+		err = rows.Scan(&isSuperUser)
+		if err != nil {
+			return false, fmt.Errorf("scanning row for superuser: %w", err)
+		}
+	}
```

Comment:
Add a else case here, basically if we dont get a row then the error message should indicate that instead of return isSuperUser as false.

Reply:
```
else {
        // Handle the case where no result is returned
        return false, fmt.Errorf("no current user found in pg_roles")
    }
```

Reply:
Done